libraries/AP_RCTelemetry/AP_VideoTX.cpp

```diff
@@ -61,9 +61,16 @@ const AP_Param::GroupInfo AP_VideoTX::var_info[] = {
     // @DisplayName: Video Transmitter Options
     // @Description: Video Transmitter Options.
     // @User: Advanced
-    // @Bitmask: 0:Pitmode
+    // @Bitmask: 0:Pitmode disabled,1: PitMode enabled
     AP_GROUPINFO("OPTIONS",  6, AP_VideoTX, _options, 0),
 
+    // @Param: LOCKING
+    // @DisplayName: Video Transmitter Locked status
+    // @Description: Video Transmitter Locked status.
+    // @User: Advanced
+    // @Bitmask: 1:Unlocked, 0 Locked
+    AP_GROUPINFO("LOCKING",  7, AP_VideoTX, _locking, 1),
```

Comment:
Just add this as a bit to VTX_OPTIONS - no need to create another parameter

Reply:
Resolved using options.